### DISCUSSION TITLE
搞这个绳网用户忙活半天，好在一切都是值得的。

### DISCUSSION THREAD

**Discussion by yulexiaotianlang** [share121/inter-knot#5576] [公告]
![190cd9c051624e4b7c7c0a8ed2792ba9_1](https://github.com/user-attachments/assets/16ed4d3d-9eee-472a-8819-1086f1edf359)
从早上一直再弄，我对英语完全一窍不通，好在终于是弄好了，终究吃了没文化的亏。
好在各位绳网大佬发的一些帖子，讲的都很细致让我也是成功登上了账号可以发帖子了。
世上还是好人多，（u.u）感谢大佬们！

**Comment by yulexiaotianlang** [comment]
![5d3d9ea5f1ecf78b29ccea449ec1cae](https://github.com/user-attachments/assets/e284eeb3-08ee-45e5-8789-c751e0d0e636)
终于弄好了开心了！

**Comment by Tu-Doou** [comment]
欢迎来到绳网:)

**Comment by fzdqz** [comment]
欢迎来到绳网！这里要关了
